src/data-analysis-environments.rst

```diff
@@ -0,0 +1,322 @@
+In Vega, users often need to provide a specification of the visual elements.
+Vega-Lite_ :cite:`satyanarayan:2017` is a simplified version of Vega.
+Vega-Lite automates the construction of some specifications (e.g., axis, legends, scales).
+Vega-Lite is compiled to Vega and can only express a subset of interactive visualizations in Vega.
+
+ .. _Vega: https://vega.github.io/vega/
+ .. _Vega-Lite: https://vega.github.io/vega-lite/
+
+.. container:: bib-item
+
+  .. bibliography:: data-analysis-environments.bib
+    :filter: key == 'satyanarayan:2017'
+
+Novice users may find it difficult to draw connections with the specification and the runtime state in Vega.
+Hoffswell et al. :cite:`hoffswell:2018` explored the design to visualize program variables within the code editor.
+With this in situ visualization technique, novice users reported performance improvements in both speed and accuracy.
+
+.. image:: https://idl.cs.washington.edu/static/images/figures/insitu-code-vis.png
+  :width: 400
+  :alt: In Situ Visualization in Vega
+
+.. container:: bib-item
+
+  .. bibliography:: data-analysis-environments2.bib
+    :filter: key == 'hoffswell:2018'
+
+Tools like Vega support the quick generation of visual representations, but they may require users to have expert programming skills in order to achieve the custom graphical design.
+The project DearData_ creatively brings the view that people use data not only to become more efficient, but also become more humane.
+It calls people to collect and hand drawing their personal data on postcards.
+In this way, people have more freeform expressions of the visual elements, but they have to draw every single data points manually.
+The project DataInk :cite:`xia:2018` manages to bridge the gap between traditional visualization tools and freeform hand drawing.
+It allows users to create expressive data visualizations on a digital canvas through direct pen and touch input.
+It enables users to bind data attributes with visual properties (e.g., shape, color) on the design.
+The research team evaluated DataInk with 8 designers and non-experts.
+The results have shown that this tool encourages users to creatively generate whimsical and personal data visualizations.
+
+.. _DearData: http://www.dear-data.com/theproject
+
+.. raw:: html
+
+  <div style="position: relative; height: 0; overflow: hidden; max-width: 100%; height: auto;">
+  <iframe width="678" height="381" src="https://www.youtube.com/embed/xlVZKGClcC0?list=TLPQMTUxMTIwMTl9VIVPBNOdFQ" frameborder="0" allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+  </div>
+
+.. container:: bib-item
+
+  .. bibliography:: data-analysis-environments2.bib
+    :filter: key == 'xia:2018'
+
+Statistical Analysis
+--------------------
+Statistical analysis is the process to generate mathematically rigorous evaluations from data.
```

Comment:
What is an "evaluation"? Consider finding a definition of "statistical analysis" from a primary source, e.g. a textbook or paper.